scripts/datasets/machine_translation/wmt2014_ende.sh

```diff
@@ -23,7 +23,7 @@ nlp_preprocess clean_tok_para_corpus --src-lang ${SRC} \
                       --src-corpus train.raw.${SRC} \
                       --tgt-corpus train.raw.${TGT} \
                       --min-num-words 1 \
-                      --max-num-words 100 \
+                      --max-num-words 250 \
```

Comment:
I think 250 is the truncation length after BPE tokenization. Have you verified that we should truncate to 250 before the BPE tokenization?

Reply:
250 is consistent with  https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/Translation/Transformer/examples/translation/prepare-wmt14en2de.sh#L181-L182

Reply:
it should be 100